File: Web/RMA-Web-Shell/projects/medicare/src/app/preauth-manager/models/preauth-levelofcare.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class PreAuthLevelOfCare extends BaseClass
{
    preAuthLevelOfCareId: number;
    levelOfCare: string;
    preAuthBreakdownId: number;
    levelOfCareId: number;
    dateTimeAdmitted: Date;
    dateTimeDischarged: Date;
    lengthOfStay: number;
    tariffCode: string;
    name: string;
    description: string;
}

File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/PreAuthLevelOfCare.cs
using RMA.Common.Entities;

using System;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class PreAuthLevelOfCare : AuditDetails
    {
        public int PreAuthLevelOfCareId { get; set; }
        public int PreAuthBreakdownId { get; set; }
        public int LevelOfCareId { get; set; }
        public DateTime DateTimeAdmitted { get; set; }
        public DateTime DateTimeDischarged { get; set; }
        public decimal LengthOfStay { get; set; }
        public string TariffCode { get; set; }
        public string LevelOfCare { get; set; }
    }
}
